source/shoh/src/threads/rotary/Rotary.cpp

```diff
@@ -12,18 +12,26 @@
 
 static QueueHandle_t * p_rotary_isr_q;
 
+static DigitalIoPin * p_sigB;
+
 extern "C" 
 {
   void
   PIN_INT0_IRQHandler (void)
   {
 	Chip_PININT_ClearIntStatus (LPC_PININT, PININTCH (PIN_INT0_IRQn));
 	portBASE_TYPE xHigherPriorityWoken = pdFALSE;
-	uint8_t data = ThreadCommon::RotaryAction::Right;
+	uint8_t data;
+
+	if (p_sigB->read())
+		data = ThreadCommon::RotaryAction::Left;
+	else
+		data = ThreadCommon::RotaryAction::Right;
+
 	xQueueSendFromISR (*p_rotary_isr_q, &data,  &xHigherPriorityWoken);
 	portEND_SWITCHING_ISR(xHigherPriorityWoken);
   }
-
+/*
```

Comment:
why you didn't delete this?

Reply:
To have it before the eyes when we will move to pcb. Then we might use dev board with buttons again.
Cleanup always can be done later.